lib/globus_client/identity.rb

```diff
@@ -43,12 +48,8 @@ def lookup_identity
 
     def extract_id(data)
       identities = data["identities"]
-      # Select identity with "used" or "private" status
-      matching_users = identities.select { |id| id["username"] == @email }
-      active_users = matching_users.select { |user| (user["status"] == "used" || user["status"] == "private") }
-      raise "No matching active Globus user found for #{@email}." if active_users.empty?
-
-      active_users.first["id"]
+      matching_user = identities.select { |id| id["username"] == @email }
+      {id: matching_user.first["id"], status: matching_user.first["status"]}
```

Comment:
Is `matching_user` here a hash or some other instance? If a hash, we could consider returning it as is.

Reply:
Updated to do this.